eng/common/templates/job/onelocbuild.yml

```
@@ -4,14 +4,15 @@ parameters:
 
   # Optional: A defined YAML pool - https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=vsts&tabs=schema#pool
   pool:
-    vmImage: windows-2019
+    vmImage: vs2017-win2016
```

I don't think this line is right per #8124.

---

Good catch, I'll change it.

```suggestion
    vmImage: windows-2019
```